src/ontodia/workspace/workspaceMarkup.tsx

```diff
@@ -46,12 +49,21 @@ export class WorkspaceMarkup extends React.Component<Props, void> {
     classTreePanel: HTMLElement;
     linkTypesPanel: HTMLElement;
     paperArea: PaperArea;
-
+    model: DiagramModel;
+    view: DiagramView;
+    private tree: ClassTree;
+    private linksToolbox: LinkTypesToolboxShell;
     private untilMouseUpClasses: string[] = [];
+    constructor(props: Props) {
+        super(props);
+        this.model = this.props.model;
+        this.view = this.props.view;
```

Comment:
Try not to assign props values to class fields. Instead, just use `this.props.view` directly.

This is similar to assigning props to intial state (except without update tracking) which could [lead to issues](https://medium.com/@justintulk/react-anti-patterns-props-in-initial-state-28687846cc2e).